Packs/MicrosoftGraphApplications/Integrations/MicrosoftGraphApplications/MicrosoftGraphApplications.py

```diff
@@ -407,11 +409,12 @@ def remove_password_service_principal_command(ms_client: Client, args: dict) ->
     )
 
 
-def unlock_configuration_service_principal_command(ms_client: Client, args: dict) -> CommandResults:
+def unlock_configuration_service_principal_command(ms_client: Client, args: dict, lock: bool = False) -> CommandResults:
```

Comment:
```suggestion
def unlock_configuration_service_principal_command(ms_client: Client, args: dict, lock: bool) -> CommandResults:
```
If lock and unlock are "equal" in status (one is not preferred over the oher), let's remove the default here. Every developer will have to pass True/False, this can prevent confusion (a dev may not notice this default causes such a tremendous change in behavior)

Reply:
Done